windows/installer-source/KaliteSetupScript.iss

```diff
@@ -196,42 +193,20 @@ begin
     end;
 end;
 
-procedure Backup013Files;

```

Comment:
Delete this function. We no longer wish to copy files to a temporary location first -- it could potentially waste a lot of disk space. Instead, we just directly call the `DoGitMigrate` function below (which could have a better name?) in order to copy video files and the database from the previous app directory directly to the user's `.kalite` directory.